user/environment-variables.md

```diff
@@ -108,11 +108,15 @@ Encrypt environment variables with the public key attached to your repository us
 
 1. If you do not have the `travis` gem installed, run `gem install travis`.
 
-2. In your repository directory, run:
+2. In your repository directory:
 
-   ```bash
-   travis encrypt MY_SECRET_ENV=super_secret --add env.matrix
-   ```
+   * If you are using https://travis-ci.org, run:
+
+       ```bash
+       travis encrypt MY_SECRET_ENV=super_secret --add env.global
+       ```
+       
+   * If you are using https://travis-ci.com, see [Encryption keys -- Usage](https://docs.travis-ci.com/user/encryption-keys#usage).
```

Comment:
Can we put .com first, and instead of linking to the docs page, mention an example? 
i.e.
`travis encrypt --com SOMEVAR="secretvalue"`

Thank you!

Reply:
The procedure for .com is more complex and takes an entire section on the linked page. Are you sure an example would be an adequate replacement? I foresee users asking why they are getting an error message in response to that command...